content/en/hosting/cht/migration/helm-charts-4x-to-5x-migration.md

```diff
@@ -0,0 +1,167 @@
+# CHT 4.x to 5.x Helm Charts Migration Guide
+
+This guide covers the migration from CHT 4.x deployments using the legacy Helm charts in the [medic/helm-charts](https://github.com/medic/helm-charts) repository to the new charts introduced in CHT 5.x within the [medic/cht-core](https://github.com/medic/cht-core) repository.
+
+## Overview
+
+The Community Health Toolkit (CHT) has moved its Helm charts from a separate repository (`medic/helm-charts`) to the main CHT Core repository (`medic/cht-core`).
+
+### Key Differences
+
+#### Chart Naming
+- **Old**: `medic/cht-chart-4x`
+- **New**: `cht-chart` (local chart from cht-core repository)
+
+## Breaking Changes Summary
+
+**IMPORTANT**: The following breaking changes require updates to your existing `values.yaml` files.
+
+### Critical Breaking Changes (Must Fix)
+
+#### 1. Field Name Changes
+
+| Old (4.x) | New (5.x) | Impact |
+|-----------|-----------|---------|
+| `clusteredCouch_enabled` | `couchdb.clusteredCouchEnabled` | **CRITICAL** - Must be updated |
+| `local.diskPath` | `local_storage.preExistingDiskPath-1` | **CRITICAL** - Must be updated |
+
+#### 2. New Required Fields
+
+| Field | Value | Impact |
+|-------|-------|---------|
+| `api.service.type` | `ClusterIP` | **CRITICAL** - Must be added to values file |
+| `preExistingDataAvailable` | `"true"` | **CRITICAL** - Must be set for existing deployments |
+
+#### 3. Storage Configuration Changes
+
+| Change | Impact |
+|--------|---------|
+| **Removed default storage sizes** | **CRITICAL** - Must explicitly set `couchdb.couchdb_node_storage_size` |
+| **No default persistent volume sizes** | **CRITICAL** - Must set platform-specific storage sizes |
+
+#### 4. Values File Structure Changes
+
+| Change | Impact |
+|--------|---------|
+| **Hierarchical structure** | **MEDIUM** - May need to restructure flat values files |
+| **Nested couchdb configuration** | **MEDIUM** - CouchDB settings now under `couchdb:` section |
+
+### Platform-Specific Storage Changes
+
+#### GKE (Google Kubernetes Engine)
+- **Required**: `couchdb.persistent_disk.size` (no default)
+
+#### EKS (Amazon Elastic Kubernetes Service)
+- **Required**: `ebs.preExistingEBSVolumeSize` (no default)
+
+### Minor Changes
+
+#### Storage Class Changes
+- **K3s-K3d Storage Class**: `local-storage` (4.x) → `local-path` (5.x) - **MINOR CHANGE**
+  - **Action Required**: Update K3s-K3d storage class if using defaults
+
+### Getting Help
+
+If you encounter issues during migration:
+
+1. Check the [CHT Documentation](https://docs.communityhealthtoolkit.org/)
+2. Review the [CHT Forum](https://forum.communityhealthtoolkit.org/)
+3. Open an issue in the [CHT Core Repository](https://github.com/medic/cht-core/issues)
+
+## Migration Steps
+
+### Step 1: Prepare Migration Values File
```

Comment:
consider using [native hextra steps](https://imfing.github.io/hextra/versions/v0.10/docs/guide/shortcodes/steps/) instead of level 3 (`###`) titles.